theme/colors/dark.css

```diff
@@ -28,10 +28,11 @@
 
 		/* Header bar */
 		--gnome-headerbar-background: #303030;
-		--gnome-headerbar-shade-color: color-mix(in srgb, black 36%, transparent);
+		--gnome-headerbar-shade-color: rgba(0, 0, 0, 0.9);
 
 		/* Toolbars */
 		--gnome-toolbar-icon-fill: #eeeeec;
+    --gnome-toolbar-border-color: var(--gnome-headerbar-shade-color);
```

Comment:
It has the same value that light (`--gnome-headerbar-shade-color`) so no need to define it again here.